### PR TITLE
Update set-up-common-area-phones.md

### PR DESCRIPTION
**Pull request by revaldiv**
Added a Note at lines  97-100 to include information about setup policy removing call app and dial pad not appearing.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-common-area-phones.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ The Common Area Phone license supports:
 
 For more information, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).
 
+ [!NOTE]
+> If you are using custom Setup Policies that unpins the calling app, the dial pad will not appear on the Common Area Phone.  
+
+For more information on Teams Setup Policies, see https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies
+
```
Comment (JohanFreelancer9) [inline Teams/set-up-common-area-phones.md:101]:
```suggestion
> [!NOTE]
> If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information on Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @revaldiv, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by revaldiv** [comment]
@scanum  what is the status of this?

**Comment by revaldiv** [comment]
@scanum @JohanFreelancer9  what is the status of this?

**Comment by JohanFreelancer9** [comment]
@scanum?

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/set-up-common-area-phones.md [hunk 2]
```diff
@@ -94,6 +94,9 @@ The Common Area Phone license supports:
 
 For more information, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).
 
+> [!NOTE]
+> If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information on Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```
Comment (tiburd) [inline Teams/set-up-common-area-phones.md:98]:
```suggestion
> If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information about Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```

File: Teams/set-up-common-area-phones.md [hunk 3]
```diff
@@ -94,6 +94,9 @@ The Common Area Phone license supports:
 
 For more information, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).
 
+> [!NOTE]
+> If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information about Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```
Comment (tiburd) [inline Teams/set-up-common-area-phones.md:98]:
```suggestion
> If you are using a custom setup policy that unpins the calling app, the dial pad doesn't appear on the Common Area Phone. For more information about Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```

**Pull request merged**